Makefile

```diff
@@ -5,8 +5,26 @@
 
 GITHUB_ORG := $(shell echo ${REPOSITORY} | cut -d "/" -f 2)
 GITHUB_REPO := $(shell echo ${REPOSITORY} | cut -d "/" -f 3)
+REPOSITORY := ${REPOSITORY}
+RANDOM_SEED := ${randomSeed}
+REGION := ${region}
+PROJECT_NAME := ${PROJECT_NAME}
 
-run:
+.EXPORT_ALL_VARIABLES:
+
+run: ci_setup
+# Conditionally setup GitHub Action OR CircleCI's environment for CI
```

Comment:
Not sure what's the best way to conditionally setup github or circleCI in makefile
this file isnt templated(in root dir) so I cant use template syntax to just wrap around the content